v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
@StarvingMarvin Oh, you mean the `$(runtime.cores)` bit? Sorry I missed that before.

And yes, I was aware that what I was suggesting would be a different test.